### PR TITLE
Improve pull requests template

### PR DESCRIPTION
**Pull request by mayaCostantini**
## Related Issues and Dependencies
Related to https://github.com/thoth-station/user-api/pull/1631#pullrequestreview-867726873

## This introduces a breaking change

- No

## This Pull Request implements
Remove checkboxes and add the `This should yield a new module release` section by default in the PR template

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,11 +4,16 @@
 
 ## This introduces a breaking change
 
-- [ ] Yes
-- [ ] No
+- Yes
+- No
```
Comment (codificat) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: Hmm... that change is already in the master branch (392118c4e75b509364a2c52a92f0dd5e7eda5dc9 in #29).

Can you rebase?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -4,11 +4,16 @@
...
 
 <!-- If this introduces a breaking change, please describe the impact and migration path for existing applications below. -->
 
+## This should yield a new module release
+
```
Comment (codificat) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: I would suggest to add an HTML comment with a brief explanation, similar to other parts of the template

**Review by codificat — CHANGES_REQUESTED** [review]
Thanks for the PR.

It needs a rebase. Also adding a suggestion

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (mayaCostantini) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: Thanks, this is now changed by https://github.com/thoth-station/template-project/pull/31/commits/eb07ab8ebd27eb198dcc06d288062a84e46d0594

**Comment by codificat** [comment]
/approve
thanks!

**Review by fridex — APPROVED** [review]
thanks! 👍🏻 💯

**Pull request merged**